opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base-apply.html" %}
+
+{% load wagtailcore_tags statusbar_tags %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+{% if object.round.specific.is_sealed %}
+<ul class="messages">
+    {% include "includes/message_item.html" with message="This submission is sealed " tag="warning" close=False %}
+</ul>
+{% endif %}
+<div class="admin-bar">
```

Comment:
Missing link back to the project page (only link in the page)

Reply:
Fixed and rebased.